FreeRTOS-Plus/Source/Application-Protocols/network_transport/using_mbedtls_pkcs11/using_mbedtls_pkcs11.h

```diff
@@ -148,6 +148,8 @@ typedef struct NetworkCredentials
     size_t userNameSize;             /**< @brief Size associated with #NetworkCredentials.pUserName. */
     const unsigned char * pPassword; /**< @brief String representing the password for MQTT. */
     size_t passwordSize;             /**< @brief Size associated with #NetworkCredentials.pPassword. */
+    char * pClientCertLabel;         /**< @brief String representing the PKCS #11 label for the client certificate. */
```

Comment:
Should these be const?

Reply:
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/main/platform/posix/transport/include/mbedtls_pkcs11_posix.h#L151-L152

The implementation used in the CSDK did not use const. However, [changing the cert and key after the network credentials have already been used does not seem to be supported behavior](https://github.com/FreeRTOS/FreeRTOS/blob/main/FreeRTOS-Plus/Source/Application-Protocols/network_transport/using_mbedtls/using_mbedtls.h#L134-L136), so I think it would be best to make these const.

Reply:
Resolved with commit:
https://github.com/FreeRTOS/FreeRTOS/pull/723/commits/ce246d74152f6a92f9f0b525e6dce8b5e5a7c03c

Reply:
Yeah. It's hard to say what the best option is here because the pkcs11 interface expects non-const pointers for attributes.